main.py

```diff
@@ -28,15 +29,17 @@ def main():
 
     Game.initialize()
     Garden.initialize()
+    Graveyard.initialize()
     Macguffin.initialize()
     ResearchFacility.initialize()
     Settings.initialize()
     Stats.initialize()
     Trip.initialize()
 
-    # This ensures new players have the new migrations. Preferably, we'd loop through the
-    # MIGRATIONS, but we have a circular dependency if we do, so this is the hack right now.
-    Player().completed_migrations = [1, 2, 3, 4, 5, 6, 7, 8]
+    # import MIGRATIONS here to resolve circular dependencies
+    from myning.migrations import MIGRATIONS  # pylint: disable=import-outside-toplevel
```

Comment:
Nice! This has been a pain point until now. I always forget you can import at other places